src/tests/ftest/util/ior_test_base.py

```diff
@@ -157,7 +157,8 @@ def run_ior_with_pool(self, intercept=None):
                 self.cancelForTicket("DAOS-3449")
             self.start_dfuse()
             self.ior_cmd.test_file.update(self.dfuse.mount_dir.value
-                                          + "/testfile")
+                                          + "/testfile{}".format(
```

Comment:
This looks like it should probably use os.path.join()

Reply:
Right. Will do.